NAMESPACE

```diff
@@ -1,7 +1,9 @@
 # Generated by roxygen2: do not edit by hand
 
+export(add_ws_to_parse_data)
```

Comment:
No, it seems to be fine. Should I remove them in this case and adapt the vignette?